rest/route/user_test.go

```diff
@@ -158,3 +160,95 @@ func (s *UserRouteSuite) TestSaveFeedback() {
 	s.NotEqual(time.Time{}, feedback[0].SubmittedAt)
 	s.Len(feedback[0].Questions, 1)
 }
+
+func TestPostUserPermissions(t *testing.T) {
+	assert := assert.New(t)
```

Comment:
I think we should avoid the assert overload.

Reply:
done by changing to a suite